clusterloader2/README.md

```diff
@@ -1,21 +1,62 @@
 # ClusterLoader
 
-### Running ClusterLoader
+## Running ClusterLoader
 
 To run ClusterLoader type:
 ```
-go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.json
+go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.yaml --nodes=<number_of_nodes>
 ```
-Flags kubeconfig and testconfig are necessary.
+Flags kubeconfig, testconfig and nodes are necessary.
 
-#### Flags
+### Flags
 
+#### Required
+
+These flags are required for any test to be run.
  - kubeconfig - path to the kubeconfig file.
- - testconfig - path to the test config file.
+ - testconfig - path to the test config file. This flag can be used multiple times if more than one test should be run.
+ - nodes - number of nodes in the cluster.
 
-### Vendor
+#### Optional
 
-Vendor is created using [govendor].
+ - report-dir - path to directory, where summaries files should be stored. If not specified, summaries are printed to standard log.
+
+## Tests
+
+### Test definition
+
+Tests can be written in yaml or json. They need to be parsable to this [api]. Test definition supports templating. The provided variable that can be used is ```Nodes``` which is a number of nodes in the cluster. \
+Example of a test definition can be found here: [load test].
+
+### Object template
 
+Object templates are similar to standard kubernetes object templates. The only different is that the templating has been added. Parameters can be passed from the test definition to the object template using the ```templateFillMap``` parameter. \
+Example of a template can be found here: [load rc template].
+
+## Measurement
+
+Currently available measurements are:
+- **APIResponsiveness** \
+This metric creates summary for latency and number for server api calls. Api calls are divided by resource, subresource, verb and scope.
+- **MetricsForE2E** \
+The metric gathers e2e metrics from components such as api server, controller manager, kubelet and scheduler.
```

Comment:
What kubelet? All kubelets?

What master components (in HA setups)? All? random?

Reply:
The same question for below too.

Reply:
`kubelet` replaced with `all kubelets (optional based on metric parameter)`.
`such as` removed to avoid confusion.